tests/regress/sql/test_plpgsql.sql

```diff
@@ -0,0 +1,7 @@
+CREATE SCHEMA s1;
```

Comment:
the schema should be dropped to make the creation always successful
```suggestion
-- start_ignore
DROP SCHEMA IF EXISTS s1 CASCADE;
-- end_ignore
CREATE SCHEMA s1;
```

Reply:
done